fake_switches/netconf/capabilities.py

```diff
@@ -61,6 +61,9 @@ def edit_config(self, request):
 
         return Response(etree.Element("ok"))
 
+    def commit_configuration(self, _):
+        return self.commit(_)
```

Comment:
underscore params are not to be used, if you just want a pass through i suggest using *args, **kwargs and let the other end deal with it

Reply:
I'd also be interested in knowing what changed this, do we still need the commit?